examples/hello-world/ml-to-fl/pt/README.md

```diff
@@ -171,34 +172,15 @@ The modified code can be found in [./code/cifar10_lightning_fl.py](./code/cifar1
 Then we can create the job using sag_pt template:
 
 ```bash
-nvflare job create -force -j ./jobs/lightning -w sag_pt -sd ./code/ -f config_fed_client.conf app_script=cifar10_lightning_fl.py
+nvflare job create -force -j ./jobs/lightning -w sag_pt -sd ./code/ \
+    -f config_fed_client.conf app_script=cifar10_lightning_fl.py \
+    -f config_fed_server.conf key_metric=val_acc_epoch model_class_path="lit_net.LitNet"
```

Comment:
Small thing, the quotes aren't needed for the -f config key-value pair strings right? We could change for consistency or leave as is

Reply:
Good suggestion I will fix this for consistency

Reply:
I will fix this in the next PR.